assets/presto_clp/main.py

```
@@ -0,0 +1,109 @@
+        """
+        Clears the cache within the docker container for cold run
+        """
+        self.docker_execute("sync")
+        self.docker_execute("echo 1 >/proc/sys/vm/drop_caches", check=False, shell=True)
+
+    def reset(self):
+        """
+        Removes a previously ingested dataset before ingesting a new one, must not throw error
+        when no dataset was ingested
+        """
+        os.system(f"{CLP_PRESTO_HOST_STORAGE}/sbin/stop-clp.sh -f")
```

Ruff (0.12.2)

Starting a process with a shell, possible injection detected

(S605)